tests/func/test_config.py

```diff
@@ -70,35 +69,19 @@ def test_non_existing(self):
         # FIXME check stdout/err
```

Comment:
can we create a ticket for this? and remove if from the code, please?

Reply:
Sure, https://github.com/iterative/dvc/issues/2086